<ui version="4.0" >
 <author></author>
 <comment></comment>
 <exportmacro></exportmacro>
 <class>QMainViewer</class>
 <widget class="QWidget" name="QMainViewer" >
  <property name="geometry" >
   <rect>
    <x>0</x>
    <y>0</y>
    <width>559</width>
    <height>476</height>
   </rect>
  </property>
  <property name="sizePolicy" >
   <sizepolicy>
    <hsizetype>5</hsizetype>
    <vsizetype>5</vsizetype>
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="focusPolicy" >
   <enum>Qt::ClickFocus</enum>
  </property>
  <property name="windowTitle" >
   <string>Main Viewer</string>
  </property>
  <layout class="QVBoxLayout" >
   <property name="margin" >
    <number>9</number>
   </property>
   <property name="spacing" >
    <number>0</number>
   </property>
   <item>
    <layout class="QHBoxLayout" >
     <property name="margin" >
      <number>0</number>
     </property>
     <property name="spacing" >
      <number>0</number>
     </property>
     <item>
      <spacer>
       <property name="orientation" >
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeType" >
        <enum>QSizePolicy::Expanding</enum>
       </property>
       <property name="sizeHint" >
        <size>
         <width>510</width>
         <height>25</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="mBtnRecord" >
       <property name="sizePolicy" >
        <sizepolicy>
         <hsizetype>1</hsizetype>
         <vsizetype>1</vsizetype>
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="minimumSize" >
        <size>
         <width>25</width>
         <height>25</height>
        </size>
       </property>
       <property name="maximumSize" >
        <size>
         <width>25</width>
         <height>25</height>
        </size>
       </property>
       <property name="focusPolicy" >
        <enum>Qt::NoFocus</enum>
       </property>
       <property name="toolTip" >
        <string>Record a movie</string>
       </property>
       <property name="text" >
        <string/>
       </property>
       <property name="icon" >
        <iconset>../Resources/movie.bmp</iconset>
       </property>
       <property name="checkable" >
        <bool>true</bool>
       </property>
       <property name="flat" >
        <bool>true</bool>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QFrame" name="mViewerFrame" >
     <property name="sizePolicy" >
      <sizepolicy>
       <hsizetype>7</hsizetype>
       <vsizetype>7</vsizetype>
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="frameShape" >
      <enum>QFrame::NoFrame</enum>
     </property>
     <property name="frameShadow" >
      <enum>QFrame::Plain</enum>
     </property>
     <property name="lineWidth" >
      <number>0</number>
     </property>
     <property name="midLineWidth" >
      <number>0</number>
     </property>
    </widget>
   </item>
  </layout>
  <widget class="QWidget" name="widget_2" >
   <property name="geometry" >
    <rect>
     <x>9</x>
     <y>9</y>
     <width>541</width>
     <height>458</height>
    </rect>
   </property>
  </widget>
  <widget class="QWidget" name="widget_3" >
   <property name="geometry" >
    <rect>
     <x>0</x>
     <y>0</y>
     <width>101</width>
     <height>31</height>
    </rect>
   </property>
  </widget>
  <widget class="QWidget" name="widget_4" >
   <property name="geometry" >
    <rect>
     <x>0</x>
     <y>0</y>
     <width>101</width>
     <height>31</height>
    </rect>
   </property>
  </widget>
 </widget>
 <pixmapfunction></pixmapfunction>
 <resources/>
 <connections/>
</ui>
